docs/buildah-bud.md

```diff
@@ -65,7 +65,7 @@ Limit the container's CPU usage. By default, containers run with the full
 CPU resource. This flag tell the kernel to restrict the container's CPU usage
 to the quota you specify.
 
-**--cpu-shares**=*0*
+**-c, --cpu-shares**=*0*
```

Comment:
Just wondering and not a thought or change needed for this PR.  We seem to list the short version first, then the fully specified version.  Alphabetically this throws the list out of order with this change.  I think it might be better if we list the full name first, then the short name as we do in the usage statement and as Docker does in their docs.  Currently our docs seem to follow short first, then fully specified so @pixdrift is following the right pattern.  Thoughts?

Reply:
Agree they should be switched, was following convention. Can raise a separate PR to switch them all at the same time if you like.

Reply:
@TomSweeneyRedHat, is it blocked on this?